tests/lang/core.pk

```diff
@@ -50,6 +59,17 @@ for i in 0..1000
   assert(abs(sin(i) / cos(i) - tan(i)) < threshold)
 end
 
+assert(abs(cosh(PI/2) - 2.5) < 0.01 )
+assert(abs(tanh(PI/4) - 1.0) < 0.5 )
+for i in 0..100
+  assert(abs(sinh(i) / cosh(i) - tanh(i)) <= 0.01)
```

Comment:
Use `threshold` here (defined above)  and the `.01` seems a very large margin.

Reply:
Please mark it resolved once you've actually done it, till then let it be unresolved.